src/content.rs

```diff
@@ -222,6 +228,39 @@ impl ExternalItemContainer for AuthorContainer {
     }
 }
 
+//
+// NewPost
+//
+//
+
+#[derive(Clone, Debug, Eq, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct NewPost {
+    pub title: String,
+    pub slug: String, // KONCH
```

Comment:
*clownFace*
removed in currently active server PR. Maube we can merhe it before so we open doors for new KONCHes but not for this one

Reply:
will fix in sep PR